src/test/java/org/cyclonedx/parsers/AbstractParserTest.java

```diff
@@ -549,11 +549,36 @@ void assertVulnerabilities(final Bom bom, final Version version) {
     assertEquals("https://example.com", vuln.getCredits().getOrganizations().get(0).getUrls().get(0));
 
     //Tools
-    assertEquals(1, vuln.getTools().size());
-    assertEquals("Sonatype CLI", vuln.getTools().get(0).getName());
-    assertEquals("Sonatype", vuln.getTools().get(0).getVendor());
-    assertEquals("1.131", vuln.getTools().get(0).getVersion());
-    assertEquals(1, vuln.getTools().get(0).getHashes().size());
+    if (version == Version.VERSION_15) {
```

Comment:
Other good tests that would check if this is working as expected, is loading the JSON and converting to a XML and the other way around, BomXMLGeneratorTest and BomJsonGeneratorTest you can check some examples there

Reply:
> Other good tests that would check if this is working as expected, is loading the JSON and converting to a XML and the other way around, BomXMLGeneratorTest and BomJsonGeneratorTest you can check some examples there

Great idea 👍 

IIUC, this is the [test ](https://github.com/shaikhu/cyclonedx-core-java/blob/409aa99c442bdfd5e8c8e71b5f18b7e22072e9e4/src/test/java/org/cyclonedx/BomXmlGeneratorTest.java#L183) for `BomXMLGeneratorTest` and this is the [test ](https://github.com/shaikhu/cyclonedx-core-java/blob/409aa99c442bdfd5e8c8e71b5f18b7e22072e9e4/src/test/java/org/cyclonedx/BomJsonGeneratorTest.java#L155) for `BomJsonGeneratorTest`. 

For the files `bom-1.5.xml` and `bom-1.5.json` (which are included in the above tests) the new `Tool` specification is used. See [here ](https://github.com/shaikhu/cyclonedx-core-java/blob/409aa99c442bdfd5e8c8e71b5f18b7e22072e9e4/src/test/resources/bom-1.5.json#L575) and [here](https://github.com/shaikhu/cyclonedx-core-java/blob/409aa99c442bdfd5e8c8e71b5f18b7e22072e9e4/src/test/resources/bom-1.5.xml#L441). 

I have updated the input test files for `/1.6/valid-bom-1.6.xml|json`  as for some reason they where using the deprecated `Tool` specification, which seems incorrect imo. See 409aa99c